sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -1030,6 +1022,21 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
           cwm(isa,jsa,l)
       end do 
 
+! instantaneous 3D cloud fraction
+      if ( imp_physics==11) then !JFDL MP
```

Comment:
GFDL MP

Reply:
@HuiyaChuang-NOAA Thanks for catching it.